### PR TITLE
Renames

### PR DESCRIPTION
**Pull request by TheTrunk**

### CODE REVIEW

File: HomeUI/src/views/explorer/Transaction.vue [hunk 1]
```diff
@@ -178,43 +178,43 @@
...
       <list-entry
-        v-if="transaction.type === 'Confirming a zelnode'"
+        v-if="transaction.type === 'Confirming a FluxNode'"
         title="Update Type"
         :number="transaction.update_type"
         classes="skinny-list-entry"
       />
       <list-entry
-        v-if="transaction.type === 'Confirming a zelnode'"
+        v-if="transaction.type === 'Confirming a FluxNode'"
```
Comment (Cabecinha84) [inline HomeUI/src/views/explorer/Transaction.vue:205]: Not sure about this one.... :D

**Review by Cabecinha84 — CHANGES_REQUESTED** [review]
I think we can change even more as daemon since version 6 have all the rcps methods (used on fluxOs) duplicated both with zel and flux names:
== Fluxnode ==
createfluxnodekey
createzelnodekey
fluxnodecurrentwinner
getdoslist
getfluxnodecount
getfluxnodeoutputs
getfluxnodestatus
getmigrationcount
getstartlist
getzelnodecount
getzelnodeoutputs
getzelnodestatus
listfluxnodeconf ( "filter" )
listfluxnodes ( "filter" )
listzelnodeconf ( "filter" )
listzelnodes ( "filter" )
startdeterministicfluxnode alias_name lockwallet
startdeterministiczelnode alias_name lockwallet
startfluxnode "all|alias" lockwallet ( "alias" )
startzelnode "all|alias" lockwallet ( "alias" )
viewdeterministicfluxnodelist ( "filter" )
viewdeterministiczelnodelist ( "filter" )
zelnodecurrentwinner

File: HomeUI/src/views/explorer/Transaction.vue (hunk 1 above)
Comment (TheTrunk) [reply HomeUI/src/views/explorer/Transaction.vue:205]: Checking :+1:

Comment (TheTrunk) [reply HomeUI/src/views/explorer/Transaction.vue]: Capitalization fixed. Actually current home production ui has it wrongly and cant show the data. As this type is obtained from daemon where it its now referenced already as 'fluxnode'

**Review by Cabecinha84 — APPROVED** [review]
Ack

**Pull request merged**